barretenberg/cpp/src/barretenberg/polynomials/polynomial.cpp

```diff
@@ -208,7 +270,7 @@ void Polynomial<Fr>::fft(const EvaluationDomain<Fr>& domain)
     ASSERT(in_place_operation_viable(domain.size));
     zero_memory_beyond(domain.size);
 
-    polynomial_arithmetic::fft(coefficients_, domain);
+    polynomial_arithmetic::fft(data(), domain);
```

Comment:
It would be interesting to see if we can just delete all of these FFT methods. None of them should be used by Honk so it might just be that easy

Reply:
Yeah we can, I meant to, done